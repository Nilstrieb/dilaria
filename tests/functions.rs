mod common;

run_test!(
    single_call,
    r#"
fn test() {
    print "correct";
}

test();
"#
);

run_test!(
    single_call_expect_return,
    r#"
fn test() {
    print "correct1";
}

test();

print "correct2";
"#
);

run_test!(
    multiple_calls,
    r#"
fn test1() {
    print "correct1";
}

fn test2() {
    print "correct2";
}

test1();
test2();

print "correct3";
"#
);
