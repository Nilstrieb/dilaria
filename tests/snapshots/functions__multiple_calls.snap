---
source: tests/functions.rs
expression: output
---
"correct1\ncorrect2\ncorrect3\n"
